flow-typed/npm/@babel/cli_vx.x.x.js

```diff
@@ -0,0 +1,90 @@
+// flow-typed signature: 9c43898047dc1cf53f5aad40011db318
```

Comment:
I think we can leave it as it is now, it will not make any difference if I will remove some of them :) looks like we did not update it for 2 years and i hope we will move the codebase to typescript during the year :)

Reply:
Ok, let's do it as part of the TypeScript rewrite then.